python/ray/train/v2/_internal/execution/scaling_policy/fixed.py

```diff
@@ -9,7 +9,7 @@
 
 class FixedScalingPolicy(ScalingPolicy):
     def make_decision_for_non_running_worker_group(
-        self, worker_group_status: WorkerGroupStatus
+        self,
```

Comment:
Add TODO about changing the ScalingPolicy API to handle certain train controller states rather than "running" vs "non running"